src/app/login/login.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, OnInit } from '@angular/core';
+import { Service1Service } from '../service1.service';
+import {Router} from '@angular/router';
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent implements OnInit {
+
+  constructor(private router:Router,private _Service1Service:Service1Service) { }
+msg;notlogin
```

Comment:
move these variable's to top and add proper type
```suggestion
msg: string;
notlogin: boolean;
```